
A bit ugly - but the build style of this 2.4 ALSA kernal stuff is equally
ugly ...

  - Rene Rebe <rene@rocklinux.org>

--- alsa-driver-1.0.4/ppc/pmac.c.orig	2004-05-05 09:36:47.000000000 +0200
+++ alsa-driver-1.0.4/ppc/pmac.c	2004-05-05 09:58:56.000000000 +0200
@@ -67,6 +67,8 @@
 	48000, 44100
 };
 
+static void snd_pmac_beep_stop(pmac_t *chip);
+
 /*
  * allocate DBDMA command arrays
  */
